lib/parsers/blog-parser.ts

```diff
@@ -0,0 +1,26 @@
+import * as path from 'path'
+import * as fs from 'fs'
+import { IParseFile } from '../interfaces'
+const cleanDeep = require('clean-deep')
+
+export async function parseBlogFile(file: IParseFile) {
```

Comment:
Why is this `async`? I suppose you could use `fs.promises.readFile` below.

Reply:
`async` used because previously used `hubdown` for converting markdown to html. Solution with using `fs.promises.readFile` looks good to save the `Promises.all`. 🌻